.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
Congratulations 🎊, no linters errors
here's what you've learn about Linters and why is really important 
[click here for more deatails](https://sourcelevel.io/blog/what-is-a-linter-and-why-your-team-should-use-it)